tools/protobuf_export/README.md

```diff
@@ -0,0 +1,38 @@
+# BigQuery Protobuf Export
+
+## Introduction
+When querying BigQuery, it is sometimes required to export data as protobuf bytes. 
+This can be useful for coding purposes, where we may want an elaborate data
+type to be exported.
+
+## Procedure
+1. Clone this reposiory.
+2. Add your proto files under `protos` folder.
+3. Run `npm install`.
+4. Run `npx webpack --config webpack.config.js --stats-error-details`.
+5. Copy the file from dist folder (`dist/pbwrapper.js`) into GCS.
+6. Use the created script in a BigQuery User Defined Function like so:
+
+```
+CREATE FUNCTION
+  <namespace>.toMyProtoMessage(input STRUCT<word STRING,
```

Comment:
namespace => dataset-id

Reply:
Add permissions:
bigquery.create_function
gcs.create_object
bigquery.read

if need job:
bigquery.job_create

bigquery needs to have:
gcs.read_object